docs/api/createI13nNode.md

```diff
@@ -27,6 +30,30 @@ var I13nAnchor = createI13nNode('a', {
 </I13nAnchor>
 ```
 
+If you have cases that you cannot replace the links with `I13n Component`, you can integrate the `scanLinks` options to get links tracked, we will not suggest to use this since it happens on client side and create additional DOM manipulation. 
```

Comment:
Adding concrete use case (e.g. dangerouslySetInnerHtml) will be helpful.

` tracked, we will not suggest` should be ` tracked.  We do not suggest`